data/frameReceiver/src/MercuryFrameDecoder - Copy.cpp

```diff
@@ -0,0 +1,775 @@
+/*

```

Comment:
Whats this copied file for?

Reply:
Yeah you are right, that should never have been added in the first place. It has been removed now.